### PR TITLE
Feat/2567 monitor oss

### LINKED ISSUE #2567: Add documentation for monitoring your InfluxDB OSS instance
I see some references scattered around the docs for the /metrics endpoint in OSS, but no content about what it is, why it exists, and what you can do with it. 

It would be nice to have a place to send people for monitoring their local influxdb instance that included information about the /metrics endpoint.

I'm thinking the following outline:

Title: Monitoring your InfluxDB Instance
- Enabling/Disabling Prometheus /metrics endpoint
- Securing the endpoint
- Summary of elements found in the endpoint
- Configuring Telegraf to scrape data from the endpoint and send it to cloud
- Configuring an OSS scraper to scrape it
- Configuring a Flux Task with promtheus.scrape to scrape it and store it locally

### LINKED ISSUE #3527: Document the OSS `/metrics` endpoint
There is a page for prometheus metrics at https://docs.influxdata.com/influxdb/v2.1/reference/prometheus-metrics/, but this seems to deal more with how to scrape/ingest prometheus metrics from an outside source, rather than documenting the metrics available on the OSS `/metrics` endpoint.

It would be good to have a list of metrics available and what they do for documentation.

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #2567
Closes #3527 

Add /metrics documentation to existing monitor (with templates) docs.

### CODE REVIEW

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 1]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP http_query_request_count Total number of query requests
+# TYPE http_query_request_count counter
+http_query_request_count{endpoint="/api/v2/query",org_id="48c88459ee424a04",status="200"} 2
+```
+
+### Query response bytes
+
+Count of bytes returned by the query.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:473]:
```suggestion
Count of bytes returned by the query endpoint.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 2]
```diff
@@ -0,0 +1,1261 @@
...
+# HELP http_query_response_bytes Count of bytes returned
+# TYPE http_query_response_bytes counter
+http_query_response_bytes{endpoint="/api/v2/query",org_id="48c88459ee424a04",status="200"} 103
+```
+
+## InfluxDB object and query statistics
+
+### Buckets total
+
+Number of total buckets on the server.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:487]:
```suggestion
Total number of buckets on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 3]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP influxdb_buckets_total Number of total buckets on the server
+# TYPE influxdb_buckets_total counter
+influxdb_buckets_total 9
+```
+
+### Dashboards total
+
+Number of total dashboards on the server.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:499]:
```suggestion
Total number of dashboards on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 4]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP influxdb_info Information about the influxdb environment.
+# TYPE influxdb_info gauge
+influxdb_info{arch="amd64",build_date="2021-12-28T22:12:40Z",commit="657e1839de",cpus="8",os="darwin",version="v2.1.1"} 1
+```
+
+### Organizations total
+
+Number of total organizations on the server.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:523]:
```suggestion
Total number of organizations on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 5]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP influxdb_organizations_total Number of total organizations on the server
+# TYPE influxdb_organizations_total counter
+influxdb_organizations_total 2
+```
+
+### Scrapers total
+
+Number of total scrapers on the server.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:535]:
```suggestion
Total number of scrapers on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 6]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP influxdb_scrapers_total Number of total scrapers on the server
+# TYPE influxdb_scrapers_total counter
+influxdb_scrapers_total 0
+```
+
+### Telegrafs total
+
+Number of total telegraf configurations on the server.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:547]:
```suggestion
Total number of Telegraf configurations on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 7]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP influxdb_telegrafs_total Number of total telegraf configurations on the server
+# TYPE influxdb_telegrafs_total counter
+influxdb_telegrafs_total 0
+```
+
+### Token services total
+
+Number of total tokens on the server.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:559]:
```suggestion
Total number of API tokens on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 8]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP influxdb_uptime_seconds influxdb process uptime in seconds
+# TYPE influxdb_uptime_seconds gauge
+influxdb_uptime_seconds{id="077238f9ca108000"} 343354.914499305
+```
+
+### Users total
+
+Number of total users on the server
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:583]:
```suggestion
Total number of users on the server.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 9]
```diff
@@ -0,0 +1,1261 @@
...
+# HELP influxdb_users_total Number of total users on the server
+# TYPE influxdb_users_total counter
+influxdb_users_total 84
+```
+
+## QC (query controller) statistics
+
+### All active
+
+Number of queries in all states
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:597]:
```suggestion
Number of queries in all states.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 10]
```diff
@@ -0,0 +1,1261 @@
...
+# TYPE query_influxdb_source_read_request_duration_seconds histogram
+query_influxdb_source_read_request_duration_seconds_bucket{op="readTagKeys",org="48c88459ee424a04",le="0.001"} 0
+--
+```
+
+## InfluxDB service statistics
+
+### Bucket service new call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:742]:
```suggestion
Number of calls to the bucket creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 11]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_bucket_new_call_total Number of calls
+# TYPE service_bucket_new_call_total counter
+service_bucket_new_call_total{method="find_bucket"} 6177
+--
+```
+
+### Bucket service new duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:755]:
```suggestion
Duration of calls to the bucket creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 12]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_bucket_new_duration Duration of calls
+# TYPE service_bucket_new_duration histogram
+service_bucket_new_duration_bucket{method="find_bucket",le="0.005"} 5876
+--
+```
+
+### Bucket service new error total
+
+Number of errors encountered.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:768]:
```suggestion
Number of errors encountered by the bucket creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 13]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_bucket_new_error_total Number of errors encountered
+# TYPE service_bucket_new_error_total counter
+service_bucket_new_error_total{code="not found",method="find_bucket_by_id"} 76
+```
+
+### Onboard service new call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:780]:
```suggestion
Number of calls to the onboarding service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 14]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_onboard_new_call_total Number of calls
+# TYPE service_onboard_new_call_total counter
+service_onboard_new_call_total{method="is_onboarding"} 11
+```
+
+### Onboard service new duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:792]:
```suggestion
Duration of calls to the onboarding service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 15]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_onboard_new_duration Duration of calls
+# TYPE service_onboard_new_duration histogram
+service_onboard_new_duration_bucket{method="is_onboarding",le="0.005"} 11
+--
+```
+
+### Organization service call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:805]:
```suggestion
Number of calls to the organization service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 16]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_org_call_total Number of calls
+# TYPE service_org_call_total counter
+service_org_call_total{method="find_labels_for_resource"} 10
+```
+
+### Organization service duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:817]:
```suggestion
Duration of calls to the organization service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 17]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_org_duration Duration of calls
+# TYPE service_org_duration histogram
+service_org_duration_bucket{method="find_labels_for_resource",le="0.005"} 10
+--
+```
+
+### Organization service new call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:830]:
```suggestion
Number of calls to the organization creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 18]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_org_new_call_total Number of calls
+# TYPE service_org_new_call_total counter
+service_org_new_call_total{method="find_org"} 1572
+--
+```
+
+### Organization service new duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:843]:
```suggestion
Duration of calls to the organization creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 19]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_org_new_duration Duration of calls
+# TYPE service_org_new_duration histogram
+service_org_new_duration_bucket{method="find_org",le="0.005"} 1475
+--
+```
+
+### Organization service new error total
+
+Number of errors encountered.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:856]:
```suggestion
Number of errors encountered by the organization creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 20]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_org_new_error_total Number of errors encountered
+# TYPE service_org_new_error_total counter
+service_org_new_error_total{code="not found",method="find_orgs"} 1
+```
+
+### Password service new call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:868]:
```suggestion
Number of calls to the password creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 21]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_password_new_call_total Number of calls
+# TYPE service_password_new_call_total counter
+service_password_new_call_total{method="compare_password"} 4
+```
+
+### Password service new duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:880]:
```suggestion
Duration of calls to the password creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 22]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_password_new_duration Duration of calls
+# TYPE service_password_new_duration histogram
+service_password_new_duration_bucket{method="compare_password",le="0.005"} 0
+--
+```
+
+### Password service new error total
+
+Number of errors encountered.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:893]:
```suggestion
Number of errors encountered by the password creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 23]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_password_new_error_total Number of errors encountered
+# TYPE service_password_new_error_total counter
+service_password_new_error_total{code="forbidden",method="compare_password"} 1
+```
+
+### Pkger service call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:905]:
```suggestion
Number of calls to the `pkger` service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 24]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_pkger_call_total Number of calls
+# TYPE service_pkger_call_total counter
+service_pkger_call_total{method="export"} 3
+```
+
+### Pkger service duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:917]:
```suggestion
Duration of calls to the `pkger` service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 25]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_pkger_duration Duration of calls
+# TYPE service_pkger_duration histogram
+service_pkger_duration_bucket{method="export",le="0.005"} 0
+--
+```
+
+### Pkger service template export
+
+Metrics for resources being exported
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:930]:
```suggestion
Metrics for exported resources.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 26]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_pkger_template_export Metrics for resources being exported
+# TYPE service_pkger_template_export counter
+service_pkger_template_export{buckets="0",by_stack="false",checks="0",dashboards="0",endpoints="0",label_mappings="0",labels="0",method="export",num_org_ids="1",rules="0",tasks="0",telegraf_configs="0",variables="0"} 3
+```
+
+### Session service call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:942]:
```suggestion
Number of calls to the session service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 27]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_session_call_total Number of calls
+# TYPE service_session_call_total counter
+service_session_call_total{method="create_session"} 3
+--
+```
+
+### Session service duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:955]:
```suggestion
Duration of calls to the session service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 28]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_session_duration Duration of calls
+# TYPE service_session_duration histogram
+service_session_duration_bucket{method="create_session",le="0.005"} 3
+--
+```
+
+### Session service error total
+
+Number of errors encountered.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:968]:
```suggestion
Number of errors encountered by the session service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 29]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_session_error_total Number of errors encountered
+# TYPE service_session_error_total counter
+service_session_error_total{code="not found",method="find_session"} 4
+```
+
+### Token service call total
+
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:980]:
```suggestion
Number of calls to the token service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 30]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_token_call_total Number of calls
+# TYPE service_token_call_total counter
+service_token_call_total{method="delete_authorization"} 3
+--
+```
+
+### Token service duration
+
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:993]:
```suggestion
Duration of calls to the token service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 31]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP service_token_duration Duration of calls
+# TYPE service_token_duration histogram
+service_token_duration_bucket{method="delete_authorization",le="0.005"} 1
+--
+```
+
+### Token service error total
+
+Number of errors encountered.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1006]:
```suggestion
Number of errors encountered by the token service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 32]
```diff
@@ -0,0 +1,1261 @@
...
+#### Example
+
+```sh
+# HELP service_token_error_total Number of errors encountered
+# TYPE service_token_error_total counter
+service_token_error_total{code="not found",method="delete_authorization"} 1
+```
+
+urm new call total
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1017]:
```suggestion
### URM new call total
Number of calls to the URM (unified resource management) creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 33]
```diff
@@ -0,0 +1,1261 @@
...
+#### Example
+
+```sh
+# HELP service_urm_new_call_total Number of calls
+# TYPE service_urm_new_call_total counter
+service_urm_new_call_total{method="find_urms"} 6451
+```
+
+urm new duration
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1028]:
```suggestion
### urm new duration
Duration of calls to the URM creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 34]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_urm_new_duration Duration of calls
+# TYPE service_urm_new_duration histogram
+service_urm_new_duration_bucket{method="find_urms",le="0.005"} 6198
+--
+```
+
+user new call total
+Number of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1040]:
```suggestion
### User new call total
Number of calls to the user creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 35]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP service_user_new_call_total Number of calls
+# TYPE service_user_new_call_total counter
+service_user_new_call_total{method="find_permission_for_user"} 4806
+--
+```
+
+user new duration
+Duration of calls.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1052]:
```suggestion
### User new duration
Duration of calls  to the user creation service.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 36]
```diff
@@ -0,0 +1,1261 @@
...
+# TYPE service_user_new_duration histogram
+service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4039
+--
+```
+
+## InfluxDB task statistics
+
+### Task executor errors counter
+
+The number of errors thrown by the executor with the type of error (ex. Invalid, Internal, etc.)
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1067]:
```suggestion
Number of errors thrown by the executor with the type of error (ex. Invalid, Internal, etc.)
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 37]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP task_executor_promise_queue_usage Percent of the promise queue that is currently full
+# TYPE task_executor_promise_queue_usage gauge
+task_executor_promise_queue_usage 0
+```
+
+### Task executor run duration
+
+The duration in seconds between a run starting and finishing.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1092]:
```suggestion
Duration (in seconds) between a task run starting and finishing.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 38]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP task_executor_run_duration The duration in seconds between a run starting and finishing.
+# TYPE task_executor_run_duration summary
+task_executor_run_duration{taskID="08017725990f6000",task_type="",quantile="0.5"} NaN
+--
+```
+
+### Task executor run latency seconds
+
+Records the latency between the time the run was due to run and the time the task started execution, by task type.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1105]:
```suggestion
Latency between the time task run's scheduled start times and actual start times, by task type.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 39]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP task_executor_run_latency_seconds Records the latency between the time the run was due to run and the time the task started execution, by task type
+# TYPE task_executor_run_latency_seconds histogram
+task_executor_run_latency_seconds_bucket{task_type="system",le="0.005"} 0
+--
+```
+
+### Task executor run queue delta
+
+The duration in seconds between a run being due to start and actually starting.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1118]:
```suggestion
Duration (in seconds) between a task run's scheduled start time and actual start time.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 40]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP task_executor_total_runs_active Total number of workers currently running tasks
+# TYPE task_executor_total_runs_active gauge
+task_executor_total_runs_active 0
+```
+
+### Task executor total runs complete
+
+Total number of runs completed across all tasks, split out by success or failure.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1143]:
```suggestion
Total number of task runs completed across all tasks, split out by success or failure.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 41]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP task_scheduler_current_execution Number of tasks currently being executed
+# TYPE task_scheduler_current_execution gauge
+task_scheduler_current_execution 128
+```
+
+### Task scheduler execute delta
+
+The duration in seconds between a run starting and finishing.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1180]:
```suggestion
Duration (in seconds) between a task run starting and finishing.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 42]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP task_scheduler_execute_delta The duration in seconds between a run starting and finishing.
+# TYPE task_scheduler_execute_delta summary
+task_scheduler_execute_delta{quantile="0.5"} NaN
+--
+```
+
+### Task scheduler schedule delay
+
+The duration between when a Item should be scheduled and when it is told to execute.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1193]:
```suggestion
Delay between when a task is scheduled to run and when it is told to execute.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 43]
```diff
@@ -0,0 +1,1261 @@
...
+```sh
+# HELP task_scheduler_schedule_delay The duration between when a Item should be scheduled and when it is told to execute.
+# TYPE task_scheduler_schedule_delay summary
+task_scheduler_schedule_delay{quantile="0.5"} NaN
+--
+```
+
+### Task scheduler total execute failure
+
+Total number of times an execution has failed.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1206]:
```suggestion
Total number of times a scheduled task execution has failed.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 44]
```diff
@@ -0,0 +1,1261 @@
...
+
+```sh
+# HELP task_scheduler_total_execute_failure Total number of times an execution has failed.
+# TYPE task_scheduler_total_execute_failure counter
+task_scheduler_total_execute_failure 0
+```
+
+### Task scheduler total execution calls
+
+Total number of executions across all tasks.
```
Comment (sanderson) [inline content/influxdb/v2.1/reference/internals/metrics.md:1218]:
```suggestion
Total number of scheduled executions across all tasks.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 45]
```diff
@@ -0,0 +1,1262 @@
...
+```sh
+# HELP task_executor_run_duration The duration in seconds between a run starting and finishing.
+# TYPE task_executor_run_duration summary
+task_executor_run_duration{taskID="08017725990f6000",task_type="",quantile="0.5"} NaN
+--
+```
+
+### Task executor run latency seconds
+
+Records the latency between the time the run was due to run and the time the task started execution, by task type.
```
Comment (jstirnaman) [inline content/influxdb/v2.1/reference/internals/metrics.md:1105]:
```suggestion
Records the latency between the time the run was due to run and the time the task started execution, by task type.
```
Latency between the task run's scheduled start time and the execution time, by task type.

Comment (jstirnaman) [inline content/influxdb/v2.1/reference/internals/metrics.md:1105]:
```suggestion
Latency between the task run's scheduled start time and the execution time, by task type.
```

File: content/influxdb/v2.1/reference/internals/metrics.md [hunk 46]
```diff
@@ -0,0 +1,1262 @@
...
+```sh
+# HELP task_executor_run_latency_seconds Records the latency between the time the run was due to run and the time the task started execution, by task type
+# TYPE task_executor_run_latency_seconds histogram
+task_executor_run_latency_seconds_bucket{task_type="system",le="0.005"} 0
+--
+```
+
+### Task executor run queue delta
+
+The duration in seconds between a run being due to start and actually starting.
```
Comment (jstirnaman) [inline content/influxdb/v2.1/reference/internals/metrics.md:1118]:
```suggestion
Duraction (in seconds) between the task run's scheduled start time and the execution time,
```

**Pull request merged**